jobs/build-mechanical.Jenkinsfile

```diff
@@ -25,10 +25,14 @@ node {
     }
 
     mechanical_streams.each{
-        echo "Triggering build for mechanical stream: ${it}"
-        build job: 'build', wait: false, parameters: [
-          string(name: 'STREAM', value: it),
-          booleanParam(name: 'EARLY_ARCH_JOBS', value: false)
-        ]
+        try {
+            echo "Triggering build for mechanical stream: ${it}"
+            build job: 'build', wait: true, parameters: [
```

Comment:
I think we can add `propagate: false` and we won't need a `try/catch` here.

https://www.jenkins.io/doc/pipeline/steps/pipeline-build-step/

Reply:
Ah, good point! That does seem like it'll be a simpler solution. 

https://github.com/coreos/fedora-coreos-pipeline/pull/1077